app/controllers/orga/community_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Orga::CommunityController < Orga::BaseController
+ 
+  def reset_user_availability
+    if User::AvailabilitySwitcher.reset
+      redirect_to orga_users_info_path, flash: { notice: 'Coaches were reset with success.' }
```

Comment:
I always try to reduce multiple exits to a single statement at the end of a method/function (unless it's a [guard clause](https://devblast.com/b/what-are-guard-clauses) in its first line). The `redirect_to` acts as a method exit in a controller. You could use the conditional branches to set the `flash[key]` messages and then redirect to `orga_users_info_path` without giving a flash kwarg _after_ the conditional block.